src/test/integration/storage/test_db_interface_admin.py

```diff
@@ -67,7 +66,7 @@ def test_remove_object_field(self):
     def test_remove_virtual_path_entries_no_other_roots(self):
         self.db_backend_interface.add_file_object(self.child_fo)
         self.assertIn(self.uid, self.db_backend_interface.file_objects.find_one(self.child_uid, {'virtual_file_path': 1})['virtual_file_path'])
-        removed_vps, deleted_files = self.admin_interface._remove_virtual_path_entries(self.uid, self.child_fo.get_uid())
+        removed_vps, deleted_files = self.admin_interface._remove_virtual_path_entries(self.uid, self.child_fo.get_uid())  # pylint: disable=protected-access
```

Comment:
`# pylint: disable=protected-access` can be moved to the top of the file so it doesn't need to be repeated for each line
IMO  this makes sense for test files (maybe we can get a separate test configuration for pylint at some point)